### PR TITLE
Adds environment setup instructions

### LINKED ISSUE #55: Doc: Add link to Bridge In Tech Setup Instructions
@mtreacy002 commented on [Sun Jun 14 2020](https://github.com/anitab-org/bridge-in-tech-web/issues/19)

## Description
As a contributor,
I need to see a link on the README file to where I can read setup instruction for development environment of Bridge In Tech backend,
so that I can contribute to the project.

## Mocks
NA
### Update [Required]
- [ ] Add a link with necessary description on README.md to go to BridgeInTech Development Environment Setup Instructions which can be found on bridge-in-tech-backend wiki repository.
## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
< 1 hour

---

@mahakbansal2019 commented on [Mon Jun 15 2020](https://github.com/anitab-org/bridge-in-tech-web/issues/19#issuecomment-643786599)

Hey @mtreacy002!
Can I work on this ?
Thanks :)

---

@mtreacy002 commented on [Mon Jun 15 2020](https://github.com/anitab-org/bridge-in-tech-web/issues/19#issuecomment-643900084)

Sure, go ahead, @mahakbansal2019 . Let me know if you need any help. Also, I'll move this issue to backend. I put it on frontend by accident. So, please add the link to the bridge-in-tech-backend repository README.md, not frontend 😉

### PR DESCRIPTION
**Pull request by mahakbansal2019**
### Description

Add link to environment setup instructions in README.md

Fixes #55 

### Type of Change:

- Documentation

**Code/Quality Assurance Only**
- This change requires a documentation update (software upgrade on readme file)

### Checklist:

- [ ] My PR follows the style guidelines of this project
- [ ] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged

### CODE REVIEW

**Review by mtreacy002 — CHANGES_REQUESTED** [review]
@mahakbansal2019, can you please place it as descriptive link just like everywhere else in the document (example here is in `Contributing` section).

You can add it as a new section in between Contributing and Branches. 



Screen Shot 2020-06-16 at 8 19 17 am

**Comment by mtreacy002** [comment]
@Menina294 and @mahakbansal2019, I should've labelled this issue a `First timer only` since this is a very simple task. But, that's ok since @mahakbansal2019 already pushed the PR and I've requested a change.

**Comment by mahakbansal2019** [comment]
@mtreacy002   
How about this ?
Screenshot from 2020-06-16 09-31-57

File: README.md
Comment (mtreacy002) [inline README.md]: It's good that you have now placed it in a separate section @mahakbansal2019. However, instead of showing it as http url, can you mask it with a descriptive link (like the one in `Contributing` section) perhaps say something like...
"To start contributing to the project, setup the backend environment on your local machine by following the instructions on the **BIT Development Environment Setup Instruction** wiki page" 
(on the bold words you can place that url link to wiki). Does this make sense?

Comment (mahakbansal2019) [reply README.md]: Okay!
Will make the desired changes :)

File: README.md
Comment (mtreacy002) [inline README.md:16]: This looks good now, @mahakbansal2019. I'm happy to approve this. We just need to wait for one more reviewer to approve the PR before it can be merged. cc @anitab-org/coding-team or @anitab-org/bridgeintech-maintainers .

Thanks again for your contribution and patience in responding to each requested changes 😉

**Review by mtreacy002 — APPROVED** [review]
LGTM 🎉

**Review by foongminwong — APPROVED** [review]
LGTM as well 👍

**Pull request merged**
